### PR TITLE
Updated Hazards (Wildfire & Water Stress)

### PR DESCRIPTION
**Pull request by jasSust**
*Added new labels and text for Wildfire in User Guide
*Added indicator of Water stress to Quickstart & User Guide

### CODE REVIEW

File: jekyll/userguide.md [hunk 1]
```diff
@@ -49,6 +57,16 @@ We implement a multi-step bias-correct procedure to ensure forward-looking cyclo
...
 
+### Water Stress Score
+
+We model current and forward-looking water stress scores using the World Resource Institute’s Aqueduct model. The state-of-the-art Aqueduct model has been extensively used by researchers in academia and industry to assess portfolio water risk.
+
+The presented water stress score is the ratio of water withdrawal to renewable water availability. Water withdrawal incorporates a diverse set of global data sources, including population, livestock, agriculture, irrigation, and other socioeconomic factors. These withdrawals encompass domestic, industrial, irrigation, and livestock uses. Renewable water availability, which includes surface and groundwater pools, is calculated using a global hydrological model that incorporates meteorological and landscape characteristics.
```
Comment (bcwaldon) [inline jekyll/userguide.md:64]: The sentence `These withdrawals encompass domestic, industrial, irrigation, and livestock uses.` seems to directly duplicate a portion of the prior sentence.

File: jekyll/userguide.md [hunk 2]
```diff
@@ -9,21 +9,29 @@ In this User Guide, we: give an overview of the outputs from climate scenario an
...
 3. **High Emissions:** This scenario covers a future where the world continues on its current trajectory (SSP5-RCP8.5). There is continued reliance on competitive markets to produce rapid technological progress.. Global markets are increasingly integrated. There is a high reliance on fossil fuels, with emissions peaking around 2090 and global mean temperatures rising approximately 5°C by 2100.
 
 ## Physical Hazard Modeling
-We report on 6 acute physical hazards at the asset level: Wildfire, drought, heatwaves, flooding, sea level rise, and tropical cyclones. The modeled results are stored as time series for each specific asset within a portfolio of assets.
+We report on 7 acute physical hazards at the asset level: wildfire, drought, heatwaves, flooding, sea level rise, and tropical cyclones. The modeled results are stored as time series for each specific asset within a portfolio of assets.
```
Comment (bcwaldon) [inline jekyll/userguide.md:20]: The list still only contains 6 elements.

Comment (bcwaldon) [inline jekyll/userguide.md:20]: The list still only contains 6 elements.

**Review by bcwaldon — CHANGES_REQUESTED** [review]

File: jekyll/userguide.md (hunk 1 above)
Comment (jasSust) [reply jekyll/userguide.md:64]: I think this is fine: made a minor tweak to: Water withdrawal incorporates a diverse set of global data sources, including: population, livestock, agriculture, irrigation, and other socioeconomic factors. They encompass domestic, industrial, irrigation, and livestock uses.

File: jekyll/userguide.md (hunk 2 above)
Comment (jasSust) [reply jekyll/userguide.md:20]: Fixed!

File: jekyll/userguide.md [hunk 3]
```diff
@@ -49,6 +57,16 @@ We implement a multi-step bias-correct procedure to ensure forward-looking cyclo
...
 
+### Water Stress Score
+
+We model current and forward-looking water stress scores using the World Resource Institute’s Aqueduct model. The state-of-the-art Aqueduct model has been extensively used by researchers in academia and industry to assess portfolio water risk.
+
+The presented water stress score is the ratio of water withdrawal to renewable water availability. Water withdrawal incorporates a diverse set of global data sources, including population, livestock, agriculture, irrigation, and other socioeconomic factors. These withdrawals encompass domestic, industrial, irrigation, and livestock uses. Renewable water availability, which includes surface and groundwater pools, is calculated using a global hydrological model that incorporates meteorological and landscape characteristics.
+
+In basins experiencing high water stress (>40%), withdrawals are large relative to renewable available water, indicating greater competition for water resources. This ratio can exceed 1.0 when non-renewable water sources like aquifers are accessed, and in practice we clip any outliers exceeding 4.0. We also interpolate values in missing years.
```
[the remark this answers is not in the document]
Comment (jasSust) [reply jekyll/userguide.md:66]: Checked w/ Tristan: changed to 0.4.

[the remark this answers is not in the document]
Comment (jasSust) [reply jekyll/userguide.md:66]: Checked w/ Tristan: changed to 0.4.

**Review by bcwaldon — APPROVED** [review]
Thanks for the updates!

**Pull request merged**